go.mod

```diff
@@ -83,7 +83,7 @@ require (
 	github.com/rancher/apiserver v0.0.0-20201023000256-1a0a904f9197
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
 	github.com/rancher/eks-operator v1.0.4
-	github.com/rancher/kubernetes-provider-detector v0.1.0
+	github.com/rancher/kubernetes-provider-detector v0.1.1-0.20201022034134-8f971d9f77cf
```

Comment:
I think we should have a new release

Reply:
https://github.com/rancher/kubernetes-provider-detector/releases/tag/v0.1.1

Reply:
@dramich Updated.